apps/sms/test/marionette/conversation_test.js

```diff
@@ -69,6 +71,72 @@ marionette('Conversation Panel Tests', function() {
         assertIsDisplayed(messagesApp.Composer.messageInput);
       });
     });
+
+
+    suite('Long MMS thread', function() {
+      function readFile(file) {
+        return require('fs').readFileSync(file, { encoding: 'base64' });
+      }
+
+      var thread;
+      var inboxView;
+      var attachmentScreenshot = readFile(
+        './apps/sms/test/marionette/assets/mms-attachment.png'
+      );
+
+      suiteTeardown(function() {
+        attachmentScreenshot = null;
+      });
+
+      setup(function() {
+        thread = ThreadGenerator.generate({
+          numberOfMessages: 50,
+          baseTimestamp: 1441114844358,
```

Comment:
> I don't exactly remember why I introduced this -- maybe because I wanted to better control the test

I think it can be useful, so let's keep it.

__nit:__ maybe use something more readable like: ```baseTimestamp: Date.UTC(2015, 8, 1, 13, 40)```?